great_expectations/expectations/core/expect_table_row_count_to_be_between.py

```diff
@@ -121,7 +121,7 @@ class ExpectTableRowCountToBeBetween(TableExpectation):
             "default_expect_table_row_count_to_be_between_rule": {
                 "variables": {
                     "false_positive_rate": 0.05,
-                    "estimator": "bootstrap",
+                    "estimator": "exact",
```

Comment:
@NathanFarmer Do we want to delete all bootstrap variables if we are changing to exact?  Otherwise, they are not used.  Thanks.

Reply:
@alexsherstinsky I was using `quantiles` here, but then was asked to change these to `exact`. If I am going to change it back due to the new `exact_estimation` parameter, it should be `quantiles` per our offline conversations regarding parity between `OnboardingDataAssistant` and Auto-Initializing Expectations.

Reply:
Per our offline conversation, we are leaving this as `exact`, but removing the config options that are not applicable.